orderer/README.md

```diff
@@ -10,13 +10,16 @@ The atomic broadcast ordering protocol for Hyperledger Fabric is described in `h
 
 * Solo ordering service (testing): The solo ordering service is intended to be an extremely easy to deploy, non-production ordering service. It consists of a single process which serves all clients, so consensus is not required as there is a single central authority.  There is correspondingly no high availability or scalability. This makes solo ideal for development and testing, but not for deployment.
 * Kafka-based ordering service (production): The Kafka-based ordering service leverages the Kafka pub/sub system to perform the ordering, but wraps this in the familiar `ab.proto` definition so that the peer orderer client code does not to be written specifically for Kafka. Kafka is currently the preferred choice for production deployments which demand high throughput and high availability, but do not require byzantine fault tolerance.
+* [//]: # (todo kafka was not removed?)
```

Comment:
Kafka was removed. delete it
Solo was removed, delete it
PBFT will not happen, change it to SmartBFT